core/src/main/java/org/apache/iceberg/encryption/EncryptionAlgorithm.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.encryption;
+
+/**
+ * Algorithm supported for file encryption.
+ */
+public enum EncryptionAlgorithm {
+    /**
+     * Counter mode (CTR) allows fast encryption with high throughput.
+     * It is an encryption only cipher and does not ensure content integrity.
+     * Inputs to CTR cipher are:
+     *  1. encryption key
+     *  2. a 16-byte initialization vector (12-byte nonce, 4-byte counter)
+     *  3. plaintext data
+     */
+    AES_CTR,
```

Comment:
We don't have this in Parquet. Do you plan to add it?

Reply:
Nope, there is no reason to add a pure CTR cipher to Parquet. This parameter will be used by other formats. We will throw an exception if this is set for Parquet files.